WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -443,6 +453,31 @@ private async Task FailTransactionSigningPhaseAsync(Round round, CancellationTok
 		}
 	}
 
+	private async Task FailFastTransactionSigningPhaseAsync(Round round, CancellationToken cancellationToken)
+	{
+		var alicesToRemove = round.Alices.Where(alice => !alice.ReadyToSign).ToHashSet();
+
+		foreach (var alice in alicesToRemove)
+		{
+			// Intentionally, do not ban Alices who have not signed, as clients using hardware wallets may not be able to sign in time.
+			Prison.Note(alice, round.Id);
```

Comment:
I'm not sure how this PR doesn't have conflict with the current `master`.
Anyhow, `Prison` got upgraded/redesigned and there is no more `Prison.Note(...)`.

If you merge `master` you will see that either we can call `Prison.FailedToConfirm` or `Prison.FailedToSign`.
None of these are perfect for this scenario.

If it's needed, I can expand the `Prison` for you to have a `Prison.FailedReadyToSign` or something similar.

Reply:
Yes, that will be necessary.

Reply:
I made this PR https://github.com/zkSNACKs/WalletWasabi/pull/11422 to expand the `Prison`.